Change links to point to new docs.tracetest.io domain (#1359)
kubeshop/tracetest: README.md

# Features

- Works out of the box with your existing OTel instrumentation, supporting [numerous backend trace datastores](https://docs.tracetest.io/supported-backends/), including Jeager and Grafana Tempo. In addition, supports adding Tracetest as an [additional pipeline](https://docs.tracetest.io/supported-backends/#using-tracetest-without-a-backend) via your OpenTelemetry Collector config. Tell us others backend datastores you want supported!
- Supporting multiple ways of creating a test, including HTTP, GRPC and Postman Collections.
- Visualize the changes you are making to your trace as you develop, enabling Observability-Driven Development.

> - [Adding assertions](https://docs.tracetest.io/adding-assertions/) based on return data from trigger call and/or data contained in the spans in your distributed trace.

Reviewer: ```suggestion
- [Add assertions](https://docs.tracetest.io/adding-assertions/) based on return data from trigger call and/or data contained in the spans in your distributed trace.
```

> - Specifying which spans to check in assertions via the [advanced selector language](https://docs.tracetest.io/advanced-selectors/).

Reviewer: ```suggestion
- Specify which spans to check in assertions via the [advanced selector language](https://docs.tracetest.io/advanced-selectors/).
```

> - Defining checks against the attributes in these spans, including properties, return status, or timing.

Reviewer: ```suggestion
- Define checks against the attributes in these spans, including properties, return status, or timing.
```

> - Tests can be created via graphical UI or via [YAML-based test definition file](https://docs.tracetest.io/test-definition-file/).

Reviewer: ```suggestion
- Create tests via graphical UI or via [YAML-based test definition file](https://docs.tracetest.io/test-definition-file/).
```

- Use the test definition file to [enable Gitops flows](https://tracetest.io/blog/integrating-tracetest-with-github-actions-in-a-ci-pipeline).
- [Tracetest CLI](https://docs.tracetest.io/command-line-tool/) allows importing & exporting tests, running tests, and more.

> - Tests are [versioned](https://docs.tracetest.io/versioning/) as the definition of the test is altered.

Reviewer: ```suggestion
- [Version tests](https://docs.tracetest.io/versioning/) as the definition of the test is altered.
```
